sso-samples/oidc-sso-sample/pickup-dispatch/src/main/webapp/home.jsp

```diff
@@ -46,13 +49,23 @@
     
     final String idToken = (String) currentSession.getAttribute("idToken");
     
-    ReadOnlyJWTClaimsSet claimsSet = null;
     String name = "";
     
+    Map<String, Object> customClaimValueMap = new HashMap<>();
+    Map<String, String> oidcClaimDisplayValueMap = new HashMap();
+    
     if (idToken != null) {
         try {
             name = SignedJWT.parse(idToken).getJWTClaimsSet().getSubject();
-            claimsSet = SignedJWT.parse(idToken).getJWTClaimsSet();
+            ReadOnlyJWTClaimsSet claimsSet = SignedJWT.parse(idToken).getJWTClaimsSet();
+            
+            ClaimManagerProxy claimManagerProxy = (ClaimManagerProxy) application.getAttribute("claimManagerProxyInstance");
+
+            customClaimValueMap = claimsSet.getCustomClaims();
+            
+            oidcClaimDisplayValueMap =
+                    claimManagerProxy.getOidcClaimDisplayNameMapping(new ArrayList<>(customClaimValueMap.keySet()));
```

Comment:
In a case of claim manager proxy not being able to fetch claim mappings, what would be the behavior ?